rest-api-spec/build.gradle

```diff
@@ -158,16 +143,13 @@ tasks.named("yamlRestCompatTest").configure {
     'get_source/81_missing_with_types/Missing document with ignore',
     'get_source/86_source_missing_with_types/Missing document source with catch',
     'get_source/86_source_missing_with_types/Missing document source with ignore',
-    'indices.clone/10_basic/Clone index via API',
     'indices.create/10_basic/Create index without soft deletes',
     // 5 below await retrofitting Removes typed URLs from mapping APIs #41676
     'indices.create/11_basic_with_types/Create index with mappings',
     'indices.create/20_mix_typeless_typeful/Create a typed index while there is a typeless template',
     'indices.create/20_mix_typeless_typeful/Create a typeless index while there is a typed template',
     'indices.create/20_mix_typeless_typeful/Implicitly create a typed index while there is a typeless template',
     'indices.create/20_mix_typeless_typeful/Implicitly create a typeless index while there is a typed template',
-    //
```

Comment:
I use the empty // to indicate that the comment above affects the section of lines
probably not the best way to comment :D

Reply:
OOOHHHHH, okay, I'll fix that. Thanks!

Reply:
b93f636